optimade/validator/validator.py

```diff
@@ -37,6 +38,24 @@ class that can be pointed at an OPTIMADE implementation and validated
 
 VERSIONS_REGEXP = r"/v[0-9]+(\.[0-9]+){,2}"
 
+__all__ = ("ImplementationValidator",)
+
+
+@dataclasses.dataclass
```

Comment:
Just a question: Is there a reason you're not using pydantic here instead?

Reply:
Uhh, guess this is slightly more lightweight but you could use either

Reply:
Was just wondering if this would solve the documentation issue as well...